docs/bench_repo_specification.md

```diff
@@ -30,27 +30,53 @@ Example:
 ./bench ../my_little_compiler/
 ```
 
-| Return code | Explanation |
-|-------------|-------------|
-| 0 | No error |
-| 1 | Internal script error |
-| 2 | Incorrect script usage |
+| Return code | Explanation            |
+|-------------|------------------------|
+| 0           | No error               |
+| 1           | Internal script error  |
+| 2           | Incorrect script usage |
 
 The script should output its results on `stdout`.
 The result is encoded in JSON.
-Output to `stderr` is shown only while benchmarking and in case of an error.
+Output to `stderr` is shown only while benchmarking (streamed to the task-detail page) and in case of an error.
```

Comment:
```suggestion
Output to `stderr` is shown only while benchmarking (streamed to the task-detail page) and in case of a *benchmark script panic* (see below).
```

or some such